docker/openchallenges/services/mcp-server-standalone.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  openchallenges-mcp-server:
+    image: ghcr.io/sage-bionetworks/openchallenges-mcp-server:latest
```

Comment:
Please remind me to show you how to create a release of the OC stack next time we meet.

Use the most recent `sha-` tag for now.